_includes/tutorials/hopping-windows/ksql/markup/dev/print-topic.adoc

```diff
@@ -4,7 +4,7 @@ Finally, let's see what's available on the underlying Kafka topic for the table.
 <pre class="snippet"><code class="sql">{% include_raw tutorials/hopping-windows/ksql/code/tutorial-steps/dev/print-topic.sql %}</code></pre>
 +++++
 
-Notice that the key for each message contains some strange characters that aren't quite printable. KSQL has combined the grouping key (movie title) with its window boundaries using a format that's not quite printable in this format. It should look something like this:
```

Comment:
nit - could you keep the «one sentence per line» rule?

Reply:
https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line